assets/dev/js/editor/controls/behaviors/scrubbing.js

```diff
@@ -0,0 +1,134 @@
+const UPDATE_VALUE = 'UPDATE-VALUE';
+const UPDATE_VALUE_ENHANCED = 'UPDATE-VALUE-ENHANCED';
```

Comment:
```suggestion
const SCRUB_REGULAR = 'UPDATE-VALUE';
const SCRUB_STEPPED = 'UPDATE-VALUE-ENHANCED';
```

Maybe something like this is more clear?

Reply:
Done, but I changed 'SCRUB_STEPPED' to SCRUB_ENHANCED.